### PR TITLE
Add Data Collection page

### PR DESCRIPTION
**Pull request by Gh0stahCZ**
Help Fragment refactored to be able to show Data Collection page.
Firebase Remote Config variable "dataCollectionMarkdown" is used for Data collection description.
My Data screen update.

### CODE REVIEW

File: app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt [hunk 1]
```diff
@@ -29,21 +29,32 @@ class HelpFragment : BaseFragment<FragmentHelpBinding, HelpVM>(R.layout.fragment
...
+        }
 
         if(arguments?.getBoolean("fullscreen") == true){
             welcome_continue_btn.visibility = View.VISIBLE
         } else {
             welcome_continue_btn.visibility = View.GONE
         }
 
-        markdown.show(help_desc, AppConfig.helpMarkdown)
+        if(arguments?.getString("type").equals("data_collection")){
```
Comment (davidvavra) [inline app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt:52]: Duplicated code, extract to method?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -168,4 +168,6 @@
     <string name="verify_phone_number">ověřit telefonní číslo</string>
     <string name="battery_saver_settings_not_found">Vypněte si prosím spořič baterie ručně.</string>
     <string name="dashboard_not_verified_secondary">Ověřte prosím své telefonní číslo, abychom vás mohli kontaktovat v případě podezření na nákazu koronavirem.</string>
+    <string name="data_collection_info">Informace o sběru dat</string>
```
Comment (davidvavra) [inline app/src/main/res/values/strings.xml:171]: All new strings should be uploaded to OneSky, translated to English and then downloaded back to this branch. Let me know your OneSky account, I will give you permissions.

File: app/src/main/kotlin/cz/covid19cz/erouska/ui/mydata/MyDataFragment.kt [hunk 3]
```diff
@@ -31,6 +31,10 @@ class MyDataFragment :
         super.onViewCreated(view, savedInstanceState)
         enableUpInToolbar(false)
         setupTabs()
+
+        data_collection_info.setOnClickListener {
+            navigate(R.id.action_nav_my_data_to_nav_help, Bundle().apply { putString("type", "data_collection") })
```
Comment (davidvavra) [inline app/src/main/kotlin/cz/covid19cz/erouska/ui/mydata/MyDataFragment.kt:36]: Why not to use SafeArgs plugin?

Comment (Gh0stahCZ) [reply app/src/main/kotlin/cz/covid19cz/erouska/ui/mydata/MyDataFragment.kt:36]: we didn't use it anywhere in the project yet
but no problem, I can add it

File: app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt (hunk 1 above)
Comment (Gh0stahCZ) [reply app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt]: Updated

File: app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt [hunk 4]
```diff
@@ -26,24 +30,41 @@ class HelpFragment : BaseFragment<FragmentHelpBinding, HelpVM>(R.layout.fragment
                 HelpCommandEvent.Command.GO_BACK -> goBack()
             }
         }
+
+        isFullscreen = arguments?.getBoolean("fullscreen") == true
+        type = when (arguments?.getString("type")) {
```
Comment (davidvavra) [inline app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt:35]: With safeArgs, you can extract fragment arguments in a type-safe way. It even provides delegate `by navArgs()`

Comment (Gh0stahCZ) [reply app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt:35]: Updated

**Pull request merged**